src/render/DrawContext.js

```diff
@@ -1573,5 +1578,32 @@ define([
                 " oc " + attributes.outlineColor.toHexString(true);
         };
 
+        /**
+         * A wrapper around gl.getExtension method, that also caches the extensions.
+         *
+         * @param {String} extensionName The name of the WebGL extension.
+         *
+         * @returns {Object|null} A WebGL extension object, or null if the extension is not available
+         *
+         * @throws {ArgumentError} if the name is missing or not a string
+         */
+        DrawContext.prototype.getExtension = function (extensionName) {
+            if (!extensionName) {
```

Comment:
This method being used internally only, we think that the checks of `extensionName` are superfluous. If you concur, could you remove them?

Reply:
As discussed, there are checks in internal methods throughout the framework, see https://github.com/NASAWorldWind/WebWorldWind/blob/develop/src/cache/GpuResourceCache.js#L142 for example.

Please remove only the type check and leave the null check.